src/utilities/flexbox.scss

```diff
@@ -32,8 +32,9 @@
     .flex#{$variant}-content-stretch  { align-content: stretch !important; }
 
     // Item
-    .flex#{$variant}-auto { flex: 1 1 auto !important; }
-    .flex#{$variant}-grow-0 { flex-grow: 0 !important; }
+    .flex#{$variant}-1        { flex: 1 !important; }
+    .flex#{$variant}-auto     { flex: auto !important; }
```

Comment:
Changing from `flex: 1 1 auto;` to just `flex: auto;` should have no effect since the computed value is the same:

![Screen Shot 2019-10-31 at 8 00 39 PM](https://user-images.githubusercontent.com/378023/67941581-48f66500-fc19-11e9-9c4e-7e86e2bb7cff.png)